tests/python/pants_test/util/test_contextutil.py

```diff
@@ -60,19 +71,21 @@ def test_environment_negation(self):
           self.assertEqual('False\n', output.read())
 
   def test_hermetic_environment(self):
-    self.assertIn('USER', os.environ)
-    with hermetic_environment_as(**{}):
-      self.assertNotIn('USER', os.environ)
+    with self.setup_non_hermetic_environment():
+      self.assertIn('USER', os.environ)
+      with hermetic_environment_as(**{}):
+        self.assertNotIn('USER', os.environ)
 
   def test_hermetic_environment_subprocesses(self):
-    self.assertIn('USER', os.environ)
-    with hermetic_environment_as(**dict(AAA='333')):
-      output = subprocess.check_output('env', shell=True).decode('utf-8')
-      self.assertNotIn('USER=', output)
-      self.assertIn('AAA', os.environ)
-      self.assertEqual(os.environ['AAA'], '333')
-    self.assertIn('USER', os.environ)
-    self.assertNotIn('AAA', os.environ)
+    with self.setup_non_hermetic_environment():
+      self.assertIn('USER', os.environ)
```

Comment:
This seems like something we could assert within the `self.setup_non_hermetic_environment()` method before yielding, but since these tests are specifically checking the `USER` variable, it seems reasonable to keep them explicit here.

Reply:
Removed it.